monkey/common/data/zero_trust_consts.py

```diff
@@ -0,0 +1,177 @@
+AUTOMATION_ORCHESTRATION = u"Automation & Orchestration"
+VISIBILITY_ANALYTICS = u"Visibility & Analytics"
+WORKLOADS = u"Workloads"
+DEVICES = u"Devices"
+NETWORKS = u"Networks"
+PEOPLE = u"People"
+DATA = u"Data"
+PILLARS = (DATA, PEOPLE, NETWORKS, DEVICES, WORKLOADS, VISIBILITY_ANALYTICS, AUTOMATION_ORCHESTRATION)
+
+STATUS_UNEXECUTED = u"Unexecuted"
+STATUS_POSITIVE = u"Positive"
+STATUS_INCONCLUSIVE = u"Inconclusive"
+STATUS_CONCLUSIVE = u"Conclusive"
+# Don't change order!
+ORDERED_TEST_STATUSES = [STATUS_CONCLUSIVE, STATUS_INCONCLUSIVE, STATUS_POSITIVE, STATUS_UNEXECUTED]
```

Comment:
Add comment about how they are ordered by severity (or some other word for it)

Reply:
Done